src/renderer/components/settings/field/decimal.vue

```diff
@@ -1,15 +1,17 @@
 <template>
-    <label>
-        {{ field.display || field.name }}
-        <input
-            type="number"
-            :name="field.name"
-            :value="value"
-            :min="field.min"
-            :max="field.max"
-            @change="$emit('value-change', $event.target.value)"
-        />
-    </label>
+    <div class="inline fields">
```

Comment:
Why are you grouping a single `field` inside a `field` tag? Just put the `inline` directly on the `field`.

Reply:
Just now noticed the intended point of this. the lables shoud be on their own line, rendering the fields all starting at the same horizontal spot. So the field should not be `inline`, but if you want to shorten them, use `x wide` syntax, which works on fields as if they we're grid columns.